roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -18,8 +18,8 @@ assetConfig:
 {% if 'logging_public_url' in openshift.master %}
   loggingPublicURL: {{ openshift.master.logging_public_url }}
 {% endif %}
-{% if 'metrics_public_url' in openshift.master %}
-  metricsPublicURL: {{ openshift.master.metrics_public_url }}
+{% if openshift_hosted_metrics_deploy | bool %}
+  metricsPublicURL: https://hawkular-metrics.{{ openshift.master.default_subdomain }}/hawkular/metrics
```

Comment:
@jfcoz I think I would rather still allow a user to override the metrics_public_url if they wish.

@abutcher do you think it would be enough to set the default for openshift.master.metrics_public_url to the above URL if metrics_public_url is not set? Only other thing it would affect (that I can think of), is that we may need to inject the metrics_public_url value into the metrics deployer if deploying metrics.